src/software/guides/addin-storage.md

```diff
@@ -0,0 +1,384 @@
+| [AddInId]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.Id): Must be provided when searching for anything other than Id. |
+| [Groups]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.Group): Used to define the scope of a row of Add-In data. Works the same as any other ObjectModel Entity. |
+| SelectClause (String): Used to filter the resulting rows based on the JSON content of the row. Works with the object path notation described in usage. Independent of WhereClause. |
+| WhereClause (String): Used to filter the resulting rows based on the JSON content of the row. Works with the object path and operator notation described in usage. Independent of SelectClause. |
+
+As an example, you can use the [API Runner tool](https://my.geotab.com/sdk/api/apiRunner.html) to perform GET operations that return one or more AddInData objects:
+
+**Example**  **2** : Get the emails of all customers who have an item with a price less than 15
+
+```javascript
+api.call("Get",
+{
+  "typeName": "AddInData",
+  "search": {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "selectClause": "customer.email",
+    "whereClause": "items.[].price < 15"
+  }
+});
+```
+
+This method call will return an array with a single AddInData object:
+
+```json
+"result": [
+  {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "id": "afLvRdUtXrE2D-XLwvqAgZQ"
+    "groups": [{"children": [], "id": "GroupCompanyId"}],
+    "details": "\"joe@smith.com\"",
+  }
+]
+```
+
+**Example**  **3** : Get all item names for user with the email **joe@smith.com**
+
+```javascript
+api.call("Get",
+{
+  "typeName": "AddInData",
+  "search": {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "selectClause": "items.[].name",
+    "whereClause": "customer.email = \"joe@smith.com\""
+  }
+});
+```
+This method call will return an array with multiple AddInData objects that satisfy both the select and where clauses.
+
+```json
+"result": [
+  {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "id": "afLvRdUtXrE2D-XLwvqAgZQ",
+    "groups": [{"children": [], "id": "GroupCompanyId"}],
+    "details": "bottles"
+  },
+  {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "id": "afLvRdUtXrE2D-XLwvqAgZQ",
+    "groups": [{"children": [], "id": "GroupCompanyId"}],
+    "details": "caps"
+  }
+]
+```
+**Note** : Both returned AddInData objects will have the same Id because they come from the same object in the database.
+
+**Example**  **4** : Get all data
+
+```javascript
+api.call("Get",
+{
+  "typeName": "AddInData",
+  "search": {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ"
+  }
+});
+```
+
+This method call will return an array with an AddInData object containing all the data stored in the AddInData object with the Id of a2C4ABQuLFkepPVf6-4OKAQ.
+
+```javascript
+"result": [
+  {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "id": "afLvRdUtXrE2D-XLwvqAgZQ",
+    "groups": [{"children": [], "id": "GroupCompanyId"}],
+    "details":Object  { ... }
```

Comment:
This has some extra non json properties. I think picked up from runner output.

Reply:
Fixed in 764501adf2514edddd8a2468e0a9123abd659036